EIPS/eip-1283.md

```diff
@@ -53,6 +53,8 @@ Definitions of terms are as below:
 Replace SSTORE opcode gas cost calculation (including refunds) with
 the following logic:
 
+* If *gasleft* is less then or equal 2300, fail the transaction
```

Comment:
```suggestion
* If *gasleft* is less then or equal 2300, fail the current call frame
```

I don't think failing the whole transaction is correct behavior. We could be inside a sub-call frame, while the parent is still having a lot of gas.

Reply:
Note that we should make this condition equivalent to:

```
If *gasleft* is less then or equal 2300, 2301 gas is deducted.
```